src/storage/txn/actions/commit.rs

```diff
@@ -45,18 +45,19 @@ pub fn commit<S: Snapshot>(
             // our transaction model. But a pessimistic lock will be left if the pessimistic
             // rollback request fails to send and the transaction need not to acquire this
```

Comment:
Another possible reason is duplicated stale `AcquireLock` request, even without `PessimisticRollback`?